packages/rules-external-tools/src/eslint-config.ts

```diff
@@ -1,26 +1,24 @@
+import { BollDirectory, BollFile, Logger } from "@boll/core";
 import { ESLint } from "eslint";
-import { BollFile } from "./boll-file";
-import { BollDirectory } from "./boll-directory";
-import { Logger, NullLogger } from "./logger";
 
 export interface ESLintRulesOptions {
   resolvePluginsRelativeTo?: BollDirectory;
-  logger?: Logger;
+  logger: Logger;
 }
 
 export class ESLintRules {
   private _eslint: ESLint;
   private _logger: Logger;
 
-  constructor(options?: ESLintRulesOptions) {
+  constructor(options: ESLintRulesOptions) {
     let eslintOptions = {};
 
     if (options && options.resolvePluginsRelativeTo) {
```

Comment:
Not a blocker but I do need to figure out a way to pass this resolvePluginsRelativeTo parameter as an option from some sort of configuration. I only know this because I did my testing with OOUI so I can see it working at scale and without specifying this option it would error out because it couldn't find the plugins specified in the config.

Reply:
How was this done previously?

My thought was to have the config file pass this as an option to the factory method. Currently inside `recommended/src/index.ts`, the constructor looks like:

```ts
RuleRegistryInstance.register("ESLintPreferConstRule", (l: Logger) => new ESLintPreferConstRule(l));
```

We could easily create the eslintrules object in the config, then pass a created instance directly to all of the eslint rule checks themselves.

Reply:
It was passed as a separate entry called eslintOptions or something along those lines in the ConfigDefinition. So it would be specified in the boll config. But this can be followed up on in a separate PR.